src/components/common/ui/rating-mui/Rating.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Next.js already have React under the hood, so you don't need it import the whole React.
In your case, just import `FC`.